src/main/resources/static/number.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+
+</head>
+<body>
+<p>Here should show a number with proper URL code</p>
+<img id="image">
+
+    <script>
+        const urlParams = new URLSearchParams(window.location.search);
+        const number = urlParams.get('number');
+        console.log(number);
+        image.src = `/api/number/${number}`
```

Comment:
does it still wok?

Reply:
Yes it does

Reply:
true :D

Reply:
But it's better practice to not to use global variables like that. Some good reasons are mentioned in answer to this question https://stackoverflow.com/questions/25931810/why-is-document-getelementbyid-not-needed
anyway, just use `const image = document.getElementById("image");`
and then ```image.src = `/api/number/${number}` ``` in next line

Reply:
ok